workflows/URLScan_with_Slack/workflow.spec.yaml

```
@@ -0,0 +1,33 @@
+extension: workflow
+products: ["insightconnect"]
+name: URLScan_with_Slack
+title: "URLScan with Slack"
+description: This workflow looks up a URL given via Slack command in urlscan.io and reports information on the URL back to Slack
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [threat_detection_and_response, chat, chatops, phishing, work_from_home, slack, covid]
```

We shouldn't duplicate use_cases and keywords

```suggestion
  keywords: [chat, chatops, phishing, work_from_home, slack, covid]
```